README.md

```diff
@@ -1399,6 +1399,7 @@ _Related: [Inventory Management](#inventory-management), [Resource Planning - En
 - [InvoicePlane](https://github.com/InvoicePlane/InvoicePlane) - Manage quotes, invoices, payments and customers for your small business. `MIT` `PHP`
 - [Kresus](https://kresus.org/) - Open source personal finance manager. ([Demo](https://kresus.org/en/demo.html), [Source Code](https://github.com/kresusapp/kresus)) `MIT` `Nodejs`
 - [Lago](https://www.getlago.com/) - Open-source metering and usage-based billing. ([Source Code](https://github.com/getlago/lago)) `AGPL-3.0` `Docker`
+- [OctoBot](https://www.octobot.online/) - Open-source cryptocurrency trading bot. ([Source Code](https://github.com/Drakkar-Software/OctoBot)) `GPL-3.0` `Python`
```

Comment:
Something else I by the way noticed was that the 2 following Links to the Documentation don't work:
> `Follow the [2 steps installation guide](https://www.octobot.online/executable_installation/`
> `Follow the [docker installation guide](https://www.octobot.online/docker_installation/) `

The other Documentation Links lead me to the intended Documentation Page.

Reply:
Thanks! I'll fix them